### ISSUE TITLE
Additional tags for AWS.Tools.* modules

### ISSUE THREAD

**Issue by scrthq** [aws/aws-tools-for-powershell#34]
First, AWESOME work on modularizing the AWSPowerShell module!!! Going through PowerShell Gallery, I actually found it a bit difficult to not immediately find what I was after as all of the AWS.Tools modules are returned in search results, even when scoping to a specific service like `AWS.Tools.EC2` and sorting by relevance. I knew what I was looking for, but the EC2 module was still buried on Page 5 even though it was an exact name match. 

My suggestions to improve discoverability, both of the modules themselves as well as supported platforms, would be to add the following tags to each module:

- Service Name, e.g. `EC2` to `AWS.Tools.EC2`. This should improve discoverability when searching for service-specific modules.
- `PSEdition_Desktop` so this shows up when specifically searching for modules compatible with Windows PowerShell
- `PSEdition_Core` so it shows up when specifically searching for modules compatible with PowerShell 6+
- `Windows` so it shows up when searching for modules compatible with the Windows platform
- `Linux` so it shows up when searching for modules compatible with the Linux platform
- `Mac` so it shows up when searching for modules compatible with the macOS platform

You can see here that searching returns 0 results when specifying the Operating System you're looking for compatible modules for (all checked to demo that this still doesn't return anything because of the missing tags):

![image](https://user-images.githubusercontent.com/12724445/62749804-66a19d00-ba23-11e9-882f-6983bd17b16a.png)

Also - if these are intended to be treated as prerelease, can they be versioned as prerelease versions so that they aren't discovered by someone searching for non-prerelease versioned moduled matching, say, `AWS*`?

**Comment by matteo-prosperi** [comment]
Hello,
thank you for your suggestion. I will look into adding more tags.

I think that the reason for the many results on the Gallery when searching for `AWS.Tools.EC2` or `AWS.Tools.S3` is that we use the names of these two modules in the description of every other AWSPowerShell and AWS.Tools module. If you search for AWS.Tools.EKS, for example, only that module is listed.
While this is not ideal, we want the description to help customers orient themselves between the many AWSPowerShell and AWS.Tools modules now available in the gallery.
I suppose the first result is `AWSPowerShell` because has many more historical downloads than the new module with the exact name match.
If this will end up being confusing for customers, I will remove the  `AWS.Tools.EC2` and `AWS.Tools.S3` text from the other modules description, this should solve the problem.

About marking the modules as prerelease, I tried that but PowerShell Gallery doesn't support modules which depends on other prerelease modules. I reported the issue to Microsoft [here](https://github.com/PowerShell/PowerShellGet/issues/517).
